sw/device/silicon_creator/manuf/base/BUILD

```
@@ -251,7 +252,9 @@ cc_library(
 cc_library(
     name = "tpm_perso_fw_ext",
     srcs = ["tpm_personalize_ext.c"],
+    hdrs = ["personalize_ext.h"],
     deps = [
+        ":ft_personalize_base",
```

If you make my suggested change above, then this would be `personalize_ext`, and you would not need the hdrs bit above, like so:
```suggestion
    deps = [
        ":personalize_ext",
```

---

done